srcpkgs/66/template

```diff
@@ -23,6 +24,13 @@ conf_files="/etc/66/init /etc/66/init.conf"
 
 system_accounts="_s6log"
 
+pre_configure() {
+
+	# Clean the default $PATH for service scripts
+	vsed -i "s@PATH=/usr/bin:/usr/sbin:/bin:/sbin:/usr/local/bin@PATH=/usr/bin:usr/sbin@" \
```

Comment:
```suggestion
	vsed -i "s@PATH=/usr/bin:/usr/sbin:/bin:/sbin:/usr/local/bin@PATH=/usr/bin:/usr/sbin@" \
```

Reply:
And maybe `/usr/local/bin`... Might make sense to leave as is, now that I think about it.

Reply:
In the runit supervision tree, the default $PATH is [/usr/bin:/usr/sbin](https://github.com/void-linux/void-runit/blob/master/2#L4). Is there a reason to have a different path for an s6 supervision tree in the same distribution?

Reply:
Added the missing slash, thank you @ericonr .

Reply:
Ah, sorry! Yes, it probably makes sense to leave as is, and PATH can be fixed in the login shell.